config/application.rb

```diff
@@ -18,6 +18,7 @@
 
 module FinCap
   class Application < Rails::Application
+    config.assets.paths << Rails.root.join("app", "assets", "fonts")
```

Comment:
The code looks good. Can this line be moved to config/initializers/assets.rb file instead, please?